app/Http/Controllers/SettingsController.php

```diff
@@ -1044,7 +1044,7 @@ public static function getPDFBranding()
      */
     public function getBackups()
     {
-
+        $settings = Setting::getSettings();
```

Comment:
> Thanks - we're definitely going to get people complaining that the date isn't formatted the way they selected in the settings tho. :(

I fixed that too, here I get the settings, and later I put the format accord to this,